### ISSUE TITLE
ionic scrolling header package issue in`building using `--prod` ( Please add a @NgModule annotation.)

### ISSUE THREAD

**Issue by iwillforce** [keephacking/ionic-scrolling-header#1]
When i  run: ionic build --prod command , aot build fails with error shown below.

```
[18:02:48]  ngc started ...
[18:02:57]  typescript error
        Unexpected value 'ScrollingHeaderModule in
        xxx/node_modules/ionic-scrolling-header/scrolling-header.module.d.ts' imported by the
        module 'xxxModule in xxx/xxx.module.ts'. Please add a @NgModule
        annotation.

Error: The Angular AoT build failed. See the issues above
at xxx\node_modules\@ionic\app-scripts\dist\aot\aot-compiler.js:237:55
at step (xxx\node_modules\@ionic\app-scripts\dist\aot\aot-compiler.js:32:23)
at Object.next (xxx\node_modules\@ionic\app-scripts\dist\aot\aot-compiler.js:13:53)
at fulfilled (xxx\node_modules\@ionic\app-scripts\dist\aot\aot-compiler.js:4:58)
[18:02:57]  copy finished in 10.69 s
```

**Comment by keephacking** [comment]
For now you can directly use the src files from git until this is fixed.

**Comment by iwillforce** [comment]
Okay @keephacking , i'll be in touch for new updates

**Comment by rajeshtbabu** [comment]
me also facing same at build time, plan advice how to use direct from git src file
thanks in advance

**Comment by iwillforce** [comment]
@rajeshtbabu follow this steps,
0-Download project from author repository
1-Rename "src" folder to "ionic-scrolling-header", and copy to your directive(folder)
2-Create a directive folder inside src (folder)
3-Change import in your app.module.ts, as you can see on my print: 
[https://i.stack.imgur.com/J0xnw.png](https://i.stack.imgur.com/J0xnw.png)

**Comment by keephacking** [comment]
Fixed in the version 0.2.2 .
